---
title: Abstract
order: 1
container: header
---
Method validation for biodetection technologies requires high purity test materials.  Traditional methods for evaluating test material purity, e.g. polymerase chain reaction (PCR), require prior knowledge of the contaminant identity and have a limited level of detection (LOD).  Whole genome sequencing using next generation sequencing (NGS) technology addresses these limitations by providing a non-target specific approach and a lower LOD due to the large number of reads generated in a sequencing run â€“ ten thousand to over one million reads.  Here we present a novel application of the pathoscope sequence analysis program (http://sourceforge.net/projects/pathoscope/) coupled with an expanded reference dataset to evaluate test material purity and validate material identity using simulated whole genome sequencing datasets.  To evaluate the ability of pathoscope to identify contaminants in test materials, in silico generated datasets for __Number of Datasets__ organisms from __number of genera__ different genera, __genera list__.   First, the results from the analysis of the single organism datasets using __PathoScope__ served as a baseline for the simulated contaminat analysis. Simulated datasets of 250 bp paried end reads with reads classified at the genus level provided suitable levels of specificity to identify potential test material contaminants.  To determine the method LOD, simulated reads from __Mixutres__ were mixed at known proportions. __Conclusions Summary__. The approach presented here provides a sample independent, highly sensitive, and specific method for evaluating test material purity.  With the rapid decrease in NGS costs, this approach is a viable alternative to PCR based methods for evaluating test material purity and can be applied in a high throughput manner for screening panels of test materials.