---
title: "Methods"
order: 3
---

This bioinformatics approach to measuring test material purity consists of three steps: 1. Sequencing reads are generated for the test material using shotgun sequencing. 2. The reads are mapped to a reference database. For this step individual reads are aligned to all possible locations in the genomes in the reference database.   We generated a more exhaustive database compared to the pathoscope database including all prokaryotic organisms in the Genbank database (http://www.ncbi.nlm.nih.gov/genbank/, accessed 10/18/2013). 3. The resulting mapping file is processed with pathoscope to assess dataset composition (__REF: Francis2013__). For the mapping step the bwa mem mapping algorithm was used (http://bio-bwa.sourceforge.net, __REF: Li2013c__).  This algorithm is capable of mapping reads to reference databases greater than 4 Gb in size.  The reference dataset included the chromosomes and plasmids for 2,709 bacterial genomes, including the strains used to validate the procedure.  Additionally, bwa mem can output all possible alignments for ambiguously mapped reads.  
 
To evaluate the specificity and sensitivity of the approach, simulated pure test material (single organism) as well as contaminated test material (mixtures of two organisms) were generated.  The organisms used in the study included common foodborne human pathogens and biothreat agents (Table \ref{table:org-names}).  Simulated datasets were generated using the ART sequencing read simulator __REF: Huang2012}.  The datasets were generated using the Illumina error models with paired end 75 and 250 base pair reads and 20 X mean coverage for each of the strains.

Simulated whole genome sequencing datasets for __unique orgs__ organisms from  __genera count__ different genera (Table Ref 1) were used to evaluate the method specificity. Datasets from the individual organisms were mapped to the all-bacterial genomes database using _bowtie2_.  The resulting mapping file was processed using _PathoScope_.   
   
__DataSetSummaryTable__

To evaluate method sensitivity, simulated datasets with a test material strain and a contaminant strain were generated.  Individual 250 bp paried end datasets for the test material and contaminant strains were generated as described in the specificity section with 20 X mean coverage.  The reads in the two datasets were subsampled in the following proportions: __proportions__ where the first number represents the proportion of reads randomly selected from the test material dataset and the second number represents the proportion of reads randomly selected from the contaminant dataset.  For example, a proportion of 0.5:0.5, the combined dataset included a random subsampling of 50 \% of the reads from the target organism dataset and 50 \% of the reads from the contaminant organism dataset.  This apporach simulates the proportions of cells in a test material and not the amount of DNA, assuming unbiased DNA extraction.  Datasets were generated for all pairwise comparisons so that each of the __mixstrain count__  strains was both a contaminant and test material for all __proportions__ proportions combinations, for a total of __number of contam__  datasets.  The individual organism datasets vary in the number of reads in the due to differences in genome size (__Table Contam Orgs__), such that 50 \% of one dataset may be different from the absolute number of reads than 50 \% of another dataset.   

To generate the simulated datasets, the single strain datasets were first mapped to the expanded reference database.  The full mapping files for the individual organisms were merged using samtools (http://samtools.sourceforge.net).  The resulting merged data files were downsampled to obtain the appropriate proportions of reads for each of the __contam org count__ combinations using an in-house python script.  The resulting simulated contaminated datasets were processed using _PathoScope_, and the output files were analyzed to determine the limit of detection of the methods using the statistical programing language R __REF: R__.  All of the scripts used in the study are available at https://github.com/nate-d-olson/genomic\_purity. 
